biomes/colors.yml

```diff
@@ -221,3 +226,9 @@ CALDERA_VOLCANO_BASE: 0x1f362e
 CALDERA_VOLCANO_BASE_EDGE: 0x345449
 CALDERA_VOLCANO_PIT_EDGE: 0x48855d
 CALDERA_VOLCANO_PIT: 0x5cb3aa
+
+EUCALYPTUS_FOREST: 0x5f8575
```

Comment:
Pop the cave biomes under a new cave biome category and eucalyptus forest under the temperate section, aside from that lgtm